pool/store/testsuite.go

```diff
@@ -10,17 +11,6 @@ import (
 
 // TestSuite runs a suite of tests against a store implementation.
 func TestSuite(t *testing.T, newStore func() Store) {
-	nodes := []Node{}
```

Comment:
Removed this to avoid the temptation to mutate it (and risk causing side effects across tests)